README.md

```diff
@@ -3,3 +3,55 @@
 Rust bindings to SpiderMonkey
 
 [Documentation](http://doc.servo.org/mozjs/)
+
+## Setting up your environment
+
+### Rust
+
+This project requires Rust nightly-2018-03-25 or greater. You can install this with Rustup.rs:
```

Comment:
I don't think it makes sense to repeat the rustup installation instructions here. It's more efficient to direct people to https://rustup.rs/.

Reply:
That makes sense, can we keep the nightly info though? I spent some time figuring out I needed to use nightly instead of stable.

Reply:
Yes, that's fine.